src/components/dashboard/newDashboard/checkMeApp/components/LoginComponent.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import { Button, Form, Icon, Input, message, notification } from 'antd';
+import { mockUsers } from '../mockDataForCheckMe';
+
+const formItemLayout = {
+    labelCol: {
+        xs: { span: 24 },
+        sm: { span: 9 }
+    },
+    wrapperCol: {
+        xs: { span: 24 },
+        sm: { span: 3 }
+    }
+};
+
+const LoginForm = (props: any) => {
+    const { form, callbackOnSubmit } = props;
+    const { getFieldDecorator, getFieldValue, setFieldsValue } = form;
+
+    const checkUser = () => {
+        if (
```

Comment:
you can simply write:
 const checkUser = () => Boolean( mockUsers.find(
                user => user.username === getFieldValue('username') && user.password === getFieldValue('password')
            ))